File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Map<Integer,Integer> dict = new HashMap<>();
        int i = 0;
        boolean result = false;
        while (i < nums.length && result == false){
            if (dict.containsKey(nums[i])){
                result = true;
            }
            dict.put(nums[i],1);
            System.out.println(nums[i]);
            i++;
        } 
        return result; 
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
def dic(t):
    d={}
    for i in t:
            if i not in d.keys():
                d[i]=1
            else:
                d[i]=d[i]+1
    return d
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        result = []
        d=dic(nums)
        d_sorted=list(d.values())
        d_sorted.sort(reverse = True)
        print(d_sorted)
        for i in range(k):
            for cle, valeur in d.items():
                if valeur == d_sorted[i]:
                    if cle not in result:
                        result.append(cle)
        return result
